github_api/prs.py

```diff
@@ -141,11 +141,13 @@ def get_ready_prs(api, urn, window):
             # because there seems to be a race where a freshly-created PR exists
             # in the paginated list of PRs, but 404s when trying to fetch it
             # directly
-            if get_is_mergeable(api, urn, pr_num):
+            mergeable = get_is_mergeable(api, urn, pr_num)
+            if mergeable is True:
                 label_pr(api, urn, pr_num, ["mergeable"])
                 yield pr
-            else:
+            elif mergeable is False:
                 label_pr(api, urn, pr_num, ["conflicts"])
+            # mergeable can also be nil, in which case we just skip it for now
```

Comment:
You should probably refer to it as `None` not `nil`, as that's what it's called in Python.